assembly/src/release/examples/amqp/python/readme.md

```diff
@@ -0,0 +1,26 @@
+## Overview
+
+This is an example of how to use the [Qpid Messaging API](https://qpid.apache.org/components/messaging-api/index.html) with ActiveMQ via the AMQP protocol.
```

Comment:
It's actually an example of Qpid Proton, rather than the linked Qpid Messaging API.

More specifically however, its an example of the old 'Messenger' API, which has seen little attention for some time due and wouldn't be recommended for new users. PROTON-1585 even removed the examples for it at Qpid .

I'd suggest the example actually be removed instead, or else updated to use the newer Proton reactive APIs, e.g. demonstrated by http://qpid.apache.org/releases/qpid-proton-0.17.0/proton/python/examples/index.html and http://qpid.apache.org/releases/qpid-proton-0.17.0/proton/python/book/tutorial.html